### PR TITLE
Upgrade Docusaurus to beta 10

### PR DESCRIPTION
**Pull request by ivarpruijn**
See [release notes](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-beta.10).

- Upgrades to beta 10.
- Fixes two breaking changes.

### CODE REVIEW

File: docs/src/theme/DocPaginator/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Translate, {translate} from '@docusaurus/Translate';
 import { tagLink, tagNavigation } from "@objectiv/tracker-browser";
 
 function DocPaginator(props): JSX.Element {
-  const {metadata} = props;
+  const {previous, next} = props;
```
Comment (ivarpruijn) [inline docs/src/theme/DocPaginator/index.tsx:14]: `metadata` changed to simply these two props

**Comment by sdirosa** [comment]
> See [release notes](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-beta.10).

This one is also interesting: https://github.com/facebook/docusaurus/pull/5830

**Review by sdirosa — APPROVED** [review]
Briefly tested this locally and it seems to work nicely. Perhaps easier to publish it on staging and do a full test there?

We could also rerun the broken link checker

**Comment by ivarpruijn** [comment]
> > See [release notes](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-beta.10).
> 
> This one is also interesting: [facebook/docusaurus#5830](https://github.com/facebook/docusaurus/pull/5830)

Hmm that will require some more testing, nice catch. Will deploy to staging next week to do so.

**Pull request closed without merging**